portal/src/graphql/adminapi/DisableUserDialog.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useContext, useMemo } from "react";
+import {
+  DefaultButton,
+  Dialog,
+  DialogFooter,
+  IDialogContentProps,
+  PrimaryButton,
+} from "@fluentui/react";
+import { Context, FormattedMessage } from "@oursky/react-messageformat";
+
+interface DisableUserDialogProps {
+  onDismiss: () => void;
+  userID: string;
+  username: string | null;
+}
+
+const dialogStyles = { main: { minWidth: "400px !important", minHeight: 0 } };
```

Comment:
I suppose we can use the default styling ([official recommended](https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog) not to exceed 340px in width).